<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMLObjectGeneratorWizard</class>
 <widget class="QWizard" name="XMLObjectGeneratorWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="welcome_page">
   <property name="title">
    <string>Welcome</string>
   </property>
   <property name="subTitle">
    <string>The ObjectXMLGenerator will go through a directory and make all the proper FIFE XML definitions. Make sure you select the proper object type, and that your directory, subdirectory, and images follow the proper conventions specified at the wiki.</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="type_page">
   <property name="title">
    <string>Object Type</string>
   </property>
   <property name="subTitle">
    <string>Is this a static object or an animated object?</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="static_radio_button">
      <property name="text">
       <string>Static</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="animated_radio_button">
      <property name="text">
       <string>Animated</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="object_path_page">
   <property name="title">
    <string>Object Path</string>
   </property>
   <property name="subTitle">
    <string>In which directory is the object located?</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="object_path_line_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browse_push_button">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="save_path_page">
   <property name="title">
    <string>Save Location</string>
   </property>
   <property name="subTitle">
    <string>Where would you like the generated XML file to be saved</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="save_path_line_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browse_push_button_2">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="atribute_page">
   <property name="title">
    <string>Attribute</string>
   </property>
   <property name="subTitle">
    <string>What other attributes would you like to modify?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
